<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionTab</class>
 <widget class="QWidget" name="ActionTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="actionList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/button-icons/add.png</normaloff>:/assets/images/button-icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/button-icons/remove.png</normaloff>:/assets/images/button-icons/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/button-icons/edit.png</normaloff>:/assets/images/button-icons/edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
